src/routes/Dressme.jsx

```diff
@@ -123,61 +111,75 @@ function DressMe(props) {
           <Column1>
             <TextInputAndValidation
               labelText="Name: "
-              fieldName={details.name}
-              onChange={(e) => setDetails({ ...details, name: e.target.value })}
+              fieldName="name"
+              value={details.name}
               shouldShowError={submitButtonClicked}
+              onChange={onChangeForFieldName("name")}
             />
             <TextInputAndValidation
               labelText="Last Name: "
-              fieldName={details.lastName}
-              onChange={(e) => setDetails({ ...details, lastName: e.target.value })}
+              fieldName={"lastName"}
+              value={details.lastName}
+              onChange={onChangeForFieldName("lastName")}
               shouldShowError={submitButtonClicked}
             />
             <Container>
               <TextInput
                 label="Phone Number: "
-                onChange={(e) => setDetails({ ...details, phone: e.target.value })}
+                fieldName={"phone"}
```

Comment:
you can pass the string without curly braces {}
same with the other TextInputs on this page